tests/unittest_nodes.py

```diff
@@ -1371,5 +1371,181 @@ def test():
     assert bool(node.is_generator())
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 10), reason="pattern matching was added in PY310"
+)
+class TestMatch:
+    @staticmethod
+    def test_match_simple():
+        node = builder.extract_node(
+            """
+        match status:
+            case 200:
+                pass
+            case 401 | 402 | 403:
+                pass
+            case None:
+                pass
+            case _:
+                pass
+        """
+        )
+        assert isinstance(node, nodes.Match)
+        assert isinstance(node.subject, nodes.Name)
+        assert isinstance(node.cases, list) and len(node.cases) == 4
+        assert node.subject.name == "status"
```

Comment:
Maybe this line would make more sense if it is written right after `assert isinstance(node.subject, nodes.Name)` because both lines are dealing with `subject`.